lib/postgrex/utils.ex

```diff
@@ -79,11 +79,11 @@ defmodule Postgrex.Utils do
   @spec default_opts(Keyword.t) :: Keyword.t
   def default_opts(opts) do
     opts
-    |> Keyword.put_new(:username, System.get_env("PGUSER") || System.get_env("USER"))
-    |> Keyword.put_new(:password, System.get_env("PGPASSWORD"))
     |> Keyword.put_new(:database, System.get_env("PGDATABASE"))
     |> Keyword.put_new(:hostname, System.get_env("PGHOST") || "localhost")
     |> Keyword.update(:port, normalize_port(System.get_env("PGPORT")), &normalize_port/1)
+    |> Keyword.put_new(:username, System.get_env("PGUSER") || System.get_env("USER"))
```

Comment:
@capitalist There may be additional modifications necessary to the code to make everything work. If you would accept a pull request into your branch, I can work on it.